website/cue/reference/releases/0.18.0.cue

```diff
@@ -139,6 +139,336 @@ releases: "0.18.0": {
 		  guide](/highlights/2021-11-18-0-18-0-upgrade-guide) for more.
 		"""
 
+	known_issues: [
+		"The `elasticsearch` sink incorrectly prints a message for each delivered event",
+	]
+
+	change_log: [
+		{
+			type: "feat"
+			scopes: ["remap transform", "config"]
+			description: """
```

Comment:
All of the descriptions need to be brought in line with [these conventions](https://github.com/vectordotdev/vector/tree/master/website#watch-your-indentation) lest the Markdown parser interpret them as code blocks.